nautobot/core/jobs/bulk_actions.py

```diff
@@ -242,6 +243,7 @@ def run(  # pylint: disable=arguments-differ
             _, deleted_info = bulk_delete_with_bulk_change_logging(queryset)
             deleted_count = deleted_info[model._meta.label]
         except ProtectedError as err:
+            # TODO this error message needs to be cleaner, ideally using a variant of handle_protectederror
```

Comment:
Agreed. As a starting point we should wrap `{err}` in backticks below so that it gets rendered literally.